uk.co.compucorp.civicrm.hrcore/hrcore.civix.php

```diff
@@ -292,7 +292,9 @@ function _hrcore_civix_insert_navigation_menu(&$menu, $path, $item) {
     $first = array_shift($path);
     foreach ($menu as $key => &$entry) {
       if ($entry['attributes']['name'] == $first) {
-        if (!$entry['child']) $entry['child'] = array();
+        if (!isset($entry['child'])) {
+          $entry['child'] = array();
+        }
```

Comment:
Is this a fix for that annoying notice that started appearing recently? How did you fix this manually? This file is created automatically by civix, so if you did that, there's a good chance the change will be lost when civix updates the file again.

Reply:
Is this the notice you're talking about?

```
Notice: Undefined index: child in _hrcore_civix_insert_navigation_menu() (line 295 of /repos/civihr/uk.co.compucorp.civicrm.hrcore/hrcore.civix.php).
```

It's caused by the new items added from [this commit](https://github.com/civicrm/civihr/commit/87100db7ddb42cf2c79c1d75ffb3f621cef70cb3) although I wouldn't blame those changes. It seems a wrong way of doing it in the `civix` function, if they want to add the `child` array if it doesn't exist surely it should check `isset`.

I know these files can be overwritten, but I was thinking whoever makes changes to it in the future would probably see the reason for this change and keep it. If you like I can